src/selection.js

```diff
@@ -71,6 +71,16 @@ var Selection = (function() {
     },
 
     /**
+     * Check if a link can be added.
+     * If the editableHost or a parent is a link element linking
+     * should not be possible.
+     */
+    canLinkBeSet: function() {
+      var linkParents = content.getParentsAndSelf(this.host, 'a');
```

Comment:
@peyerluk This iterates throught all parents of the dom up to the html tag.
I guess we shouldn't go further than the container in which editable.js is initialized.